bin/themis.bat

```diff
@@ -17,7 +17,12 @@ if not exist "%STARTUP_SCRIPT%" (
 )
 
 rem FIXME: Some wrong case exists in passing the argument list.
-%THEMIS_VIM% -u NONE -i NONE -n -N %THEMIS_ARGS% --cmd "source %STARTUP_SCRIPT%" -- %* 2>&1
+rem DO NOT directly output the result to command prompt while 'normal' command in Vim script
+rem will move a cursor of command prompt in that case.
+set THEMIS_LOG="%USERPROFILE%\AppData\Local\Temp\themis.log"

```

Comment:
`%TMP%` and `%TEMP%` point to `%USERPROFILE%\AppData\Local\Temp\` by default.
See also: http://www.softantenna.com/wp/windows/windows-tmp-vs-temp/